lib/src/handle_access_token_response.dart

```diff
@@ -68,9 +68,18 @@ Credentials handleAccessTokenResponse(http.Response response, Uri tokenEndpoint,
     }
 
     var expiresIn = parameters['expires_in'];
-    if (expiresIn != null && expiresIn is! int) {
-      throw FormatException(
-          'parameter "expires_in" was not an int, was "$expiresIn"');
+    if (expiresIn != null) {
+      if (expiresIn is String) {
+        try {
+          expiresIn = double.tryParse(expiresIn)!.toInt();
+        } catch (e) {
+          throw FormatException(
+              'parameter "expires_in" was not an int, and given string cant not be parsed to int : "$expiresIn"');
+        }
```

Comment:
```suggestion
        try {
          expiresIn = double.parse(expiresIn).toInt();
        } on FormatException {
          throw FormatException(
              'parameter "expires_in" was not an int, and given string cannot be parsed to int : "$expiresIn"');
        }
```

Reply:
* [ ] Typo `cant not`
 * [ ] Why not just do `on FormatException`, then we don't need the null-check operator...

Reply:
I fixed this and merged :D